ruby_03-professional_rails_applications/building-a-vps.markdown

```diff
@@ -308,7 +308,15 @@ them which application process to use when serving requests.
 ### Configuring NGINX
 
 First, we're going to need to know where to tell Passenger to look for an installed version of Ruby.
-Fortunately passenger includes a script for finding this information, which we can use like so:
+Fortunately passenger includes a script for finding this information. Prior to setting that up we'll need to know where our installation of Ruby is hiding:
+
+```sh
+which ruby
```

Comment:
During the lesson you had us check ```which ruby``` at this point to demonstrate where `/home/deploy/.rvm/*` lives, so I figured you wanted it in the lesson plan for next time. You're right though, we're using the passenger-config which is all that really matters for the configuration.